From: "Barak A. Pearlmutter" <barak+git@pearlmutter.net>
Date: Thu, 18 Jul 2019 20:21:52 +0200
Subject: no hyperref hypertex

The hypertex hyperref option package is a problem with pdflatex, and
removing it leaves the output dvi unchanged anyway.
---
 nbib.nw | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nbib.nw b/nbib.nw
index 3e2b1ba..a8d3cfd 100644
--- a/nbib.nw
+++ b/nbib.nw
@@ -3,7 +3,7 @@
 \documentclass{article}
 \usepackage{fullpage}
 \usepackage{noweb,url}
-\usepackage[hypertex]{hyperref}
+\usepackage{hyperref}
 \noweboptions{smallcode}
 
 \def\BibTeX{{\rm B\kern-.05em{\sc i\kern-.025em b}\kern-.08em
